compose/service.py

```diff
@@ -334,16 +334,21 @@ def _get_links(self, link_to_self):
 
     def _get_volumes_from(self):
         volumes_from = []
-        for volume_source in self.volumes_from:
+        for volume_from in self.volumes_from:
+            volume_source, mount_option = volume_from
```

Comment:
A namedtuple would be good, that way you could do `volume_from.source` and `volume_from.mount_option`.